torchmultimodal/modules/encoders/text_encoder.py

```diff
@@ -16,28 +16,21 @@
 from torchmultimodal.utils.attention import get_extended_attention_mask
 
 
-class TextEncoder(nn.Module):
+class BERTTextEncoder(nn.Module):
     """
-    General text transformer encoder with embeddings, similar to BERT / RoBERTa.
+    General text transformer encoder with embeddings, following BERT.
     Can be constructed with any user-provided embeddings and encoder.
 
     Based on https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py#L870
 
     Attributes:
         embeddings (nn.Module): Module that projects text token ids into embeddings.
-            ``forward()`` should follow interface:
-                input_ids: Optional[Tensor],
-                token_type_ids: Optional[Tensor],
-                position_ids: Optional[Tensor],
-                inputs_embeds: Optional[Tensor],
-        encoder (nn.Module): Module for transformer encoder. ``forward()`` should follow interface:
-            Inputs:
-                hidden_states: Tensor,
-                attention_mask: Optional[Tensor], shape [batch_size, num_heads, query_seq_length, key_seq_length]
-            Returns:
-                ``TransformerOutput``
-        layernorm (nn.Module, optional): Module for layernorm to be applied after encoder, if provided.
-        pooler (nn.Module, optional): Module for pooler to be applied after layernorm, if provided.
+            See :py:class: BERTTextEmbeddings for interface.
```

Comment:
- Add backticks around the class name

:py:class: {single_backtick}BERTTextEmbeddings{single_backtick}

- Ideally should point to the full path: `torchmultimodal.modules.layers.text_embedding.BERTTextEmbedding`

https://stackoverflow.com/questions/34052582/how-do-i-refer-to-classes-and-methods-in-other-files-my-project-with-sphinx